weather_dl/download_pipeline/parsers.py

```diff
@@ -282,15 +284,15 @@ def date_range(start: datetime.date, end: datetime.date, increment: int = 1) ->
     return (start + datetime.timedelta(days=x) for x in range(0, (end - start).days + 1, increment))
 
 
-def _parse_lists(config_parser: configparser.ConfigParser, section: str = '') -> t.Dict:
-    """Parses multiline blocks in *.cfg files as lists."""
-    config = dict(config_parser.items(section))
-
+def _parse_lists(config: dict, section: str = '') -> t.Dict:
+    """Parses multiline blocks in *.cfg and *.json files as lists."""
     for key, val in config.items():
-        if '/' in val and 'parameters' not in section:
-            config[key] = parse_mars_syntax(val)
-        elif '\n' in val:
-            config[key] = _splitlines(val)
+        # Check string type for backward compatibility. i.e. <= v0.1.7 supports "padding": 0 in json config
```

Comment:
Is this a TODO?

Reply:
Naa. It was to explain why we're checking for string type in the below line.
The message could've been better.
Updated one: **# Checks str type for backward compatibility since it also support "padding": 0 in json config**